explainer.md

```diff
@@ -0,0 +1,255 @@
+    c. A replaced element for the root/html element identified via ::shared-old(#root).
+
+6. Apply the following UA stylesheet to the pseudo elements on the new page.
+```
+::shared-old(#root), ::shared-new(#root), ::shared-container(#shared-header) {
+  position: fixed;
+  top: 0px;
+  left: 0px;
+}
+
+::shared-old(#root), ::shared-new(#root) {
+  width: 100vw;
+  height: 100vh;
+  content: element(html);
+}
+::shared-old(#root) {
+  // This is the saved output referenced in step 3.
+  content: cached-element(html);
+}
+::shared-new(#root) {
+  content: element(html);
+}
+
+::shared-old(#shared-header), ::shared-new(#shared-header) {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  
+  content: element(#header);
+}
+::shared-old(#shared-header) {
+  // This is the saved output referenced in step 3.
+  content: cached-element(#header);
+}
+::shared-new(#shared-header) {
+  content: element(#header);
+}
+
+// Update the container's size and transform to the shared element on the new page.
+::shared-container(#shared-header) {
+  width: 200px;
+  height: 200px;
+  transform: translate(0px, 100px);
+}
+```
+
+7. Once no pseudo element has an active animation, remove them from the top layer.
+
+## Live Animatable Properties
+A common capability desirable during transitions is to interpolate styles like border-radius that change the element's shape. The [container transform](https://material.io/design/motion/the-motion-system.html#container-transform) examples show a visual demo of that. Painting properties like the element's border within its image when using the element() function makes this difficult. The following example[^2] shows how developers can control which styles are captured in the element() function vs being applied to the pseudo container on the old page.
+
+```
+<html>
+<head>
+  <style>
+    .animated {
+      width:100px;
+      height:100px;
+      border: 10px solid black;
+      border-radius: 10% 10%;
+      box-shadow: 0px 0px 10px;
+      
+      &:transition {
+        /* Retain the border to ensure it is painted transparent but the box size is unchanged. */
+        border: 10px solid transparent;
+        border-radius: none;
+        box-shadow: none;
+      }
+    }
+    
+    ::shared-container(#shared-header) {
+      border: 10px solid black;
+      border-radius: 10% 10%;
+      box-shadow: 0px 0px 10px;
+    }
+    
+    ::shared-old(#shared-header) {
+      top: -10px;
+      left: -10px;
+    }
+  </style>
+</head>
+<body>
+  <div class="animated" shared-id="shared-header">hello world</div>
+</body>
+</html>
+```
+
+This capability requires the following changes to the steps mentioned in the design above :
+* A new pseudo class ("transition") is introduced which is activated when an element is being displayed using pseudo elements in step 1 and 5.
```

Comment:
By activated you mean it applies when the element is being displayed using pseudo elements and doesn't apply otherwise?

Also, I'll be honest I had to keep looking back at what steps 1 and 5 and 3 (below) are, it might be nicer to reference those steps but also briefly explain what they are again

Reply:
Added a comment to reiterate what the step was doing.